app/src/pages/inside/common/infoLine/infoLine.jsx

```diff
@@ -33,6 +33,9 @@ export class InfoLine extends Component {
     const passed = executions.passed / executions.total * 100;
     const failed = executions.failed / executions.total * 100;
     const skipped = executions.skipped / executions.total * 100;
+    const endTime = this.props.data.end_time;
+    const startTime = this.props.data.start_time;
+    const duration = (endTime - startTime) / 1000;
```

Comment:
It's better to use the util function `getDuration` from `common/utils/timeDateUtils.js`

Reply:
Thank you, I just spotted this util later. I'll fix.